sdk/client/clientset/versioned/typed/apis/v1alpha1/fake/fake_apiexport.go

```diff
@@ -19,11 +19,10 @@ limitations under the License.
 package fake
 
 import (
-	gentype "k8s.io/client-go/gentype"
-
 	v1alpha1 "github.com/kcp-dev/kcp/sdk/apis/apis/v1alpha1"
 	apisv1alpha1 "github.com/kcp-dev/kcp/sdk/client/applyconfiguration/apis/v1alpha1"
 	typedapisv1alpha1 "github.com/kcp-dev/kcp/sdk/client/clientset/versioned/typed/apis/v1alpha1"
+	gentype "k8s.io/client-go/gentype"
```

Comment:
I still confused why import blocks are reverted. I would expect to see order:
```
core

3rdparties

kcp-dev ones 
```

Any reason these imports now are how they are?
```

Reply:
They should be as 
```yaml
  settings:
    gci:
      sections:
        - standard
        - default
        - prefix(k8s.io)
        - prefix(github.com/kcp-dev)
        - prefix(github.com/kcp-dev/kcp)
        - blank
        - dot
      custom-order: true
```

Reply:
I think these changes are N/A anymore. This PR does not modify sdk/client anymore